conf/config.py

```diff
@@ -9,7 +9,6 @@
         ],
         'topic': 'topic',
         'msg_id': 'headers.message-id',
-        'build_id': 'body.info.id',
```

Comment:
Is there a good reason to remove the `build_id` and `build_name` fields? Seems like leaving them in the mapping shouldn't cause problems, and they may be useful in the future.

Reply:
The parsed message from mbs-messaging-umb will be converted to a dict and passed to handlers in MBS in format of `func(**kwargs)`, `build_id` and `build_name` are not used in the handlers at all, so they were removed from handler args, the change of `repo_tag` to `tag_name` is for implementing MBS task router in a more easier way,  after the change, calling handlers with these args will result in unexpected keyword argument error.

Reply:
I see, thanks for the explanation.